pkgs/development/python-modules/sanic/default.nix

```diff
@@ -3,13 +3,13 @@
 , aiofiles
 , beautifulsoup4
 , buildPythonPackage
-, doCheck ? true
+, doCheck ? !stdenv.isDarwin # on Darwin, tests fail but pkg still works
 , fetchFromGitHub
 , gunicorn
 , httptools
 , multidict
 , pytest-asyncio
- ,pytestCheckHook
+ , pytestCheckHook
```

Comment:
```suggestion
, pytestCheckHook
```

Reply:
There are actually two whitespace corrections to make - I wasn't sure whether to include them, but I've pushed them both now. Thanks for the review!!